### PR TITLE
Update webpass admin web pages

### PR DESCRIPTION
**Pull request by sreedhards**
Update webpass admin web pages

### CODE REVIEW

File: pages/web/web-paas/administration-web/guide.en-sg.md [hunk 1]
```diff
@@ -5,168 +5,26 @@ section: Administration
...
-![Service tab - App Configuration](images/app-configuration.png "0.7")
-
-The "Configuration" tab provides an overview of the application's configuration pulled from its `.platform.app.yaml` file.
-
-#### Services
-
-Each service has a tab on the left, so select the one you are interested in.
-
-![Service tab - Service Overview](images/service-overview.png "0.7")
+{{% description %}}
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-web/guide.en-sg.md]: Whatever this is supposed to do, it doesn't work.

File: pages/web/web-paas/administration-web/guide.en-sg.md [hunk 2]
```diff
@@ -5,168 +5,26 @@ section: Administration
...
 
-The overview tab gives you metadata information regarding the service. It tells you what size container it has been configured for and the amount of persistent disk given to it in your `services.yaml` file.
+![Overview of projects in the console](images/all-projects.png "0.5")
 
-![Service tab - Service Configuration](images/service-configuration.png "0.7")
+This console offers you ways to manage projects and environments other than the [command line interface (CLI))](../../development/cli/_index.md).
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-web/guide.en-sg.md]: Relative link won't work.

File: pages/web/web-paas/administration-web/guide.en-sg.md [hunk 3]
```diff
@@ -5,168 +5,26 @@ section: Administration
...
 
-The "Configuration" tab provides an overview of the service configuration that has been pulled from the `services.yaml` file.
+## Project list
 
-#### Routes
+The [main console page](https://console.platform.sh) gives you an overview of all your projects.
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-web/guide.en-sg.md]: Wrong link.

File: pages/web/web-paas/administration-web/guide.en-sg.md [hunk 4]
```diff
@@ -5,168 +5,26 @@ section: Administration
...
+You can narrow the list down by searching for specific projects or choosing a specific organization from the menu.
 
-![Service tab - Routes](images/routes.png "0.7")
+Once you select a project, you can [configure the project settings](./configure-project.md).
+You can also choose individual environments within the project and [configure each environment](./configure-environment.md).
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-web/guide.en-sg.md]: These relative links won't work.

File: pages/web/web-paas/administration-web/configure-project/guide.en-sg.md [hunk 5]
```diff
@@ -1,84 +1,64 @@
...
-> 
-
-Selecting a user will allow you to either edit that user's permissions or delete the user's access to the project entirely.
-
-![Manage users of your Web PaaS project](images/edit-user.png "0.7")
+## Certificates
 
-If you check the `Project admin` box, this user will be an administrator of the project and will have fulll access on all environments. If you uncheck the box, you'll have the option of adjusting the user's permissions on each environment.
+The *Access* section shows a list of your project's TLS certificates.
+To see details about a certificate or delete one, click **Edit {{< icon chevron >}}**.
```
Comment (bs-ovh) [inline pages/web/web-paas/administration-web/configure-project/guide.en-sg.md]: This doesn't work.

**Review by bs-ovh — CHANGES_REQUESTED** [review]

File: pages/web/web-paas/administration-web/guide.en-sg.md (hunk 1 above)
Comment (sreedhards) [reply pages/web/web-paas/administration-web/guide.en-sg.md]: updated

File: pages/web/web-paas/administration-web/configure-project/guide.en-sg.md (hunk 5 above)
Comment (sreedhards) [reply pages/web/web-paas/administration-web/configure-project/guide.en-sg.md]: corrected

File: pages/web/web-paas/administration-users/guide.en-asia.md [hunk 6]
```diff
@@ -55,24 +56,30 @@ The following table shows the available roles for environment types.
...
 >> 
->> 1. In the console, select the project where you want to add a new user.
->> 2. Under **Settings**, click **Access**.
->> 3. Click **+ Add**.
->> 4. Add the user's details and choose their permissions.
->> 5. Click **Save**.
+>> <!--This is in HTML to get the icon not to break the list. -->
+>> <ol>
+>>   <li>Select the project where you want to add a new user.</li>
+>>   <li>Click {{< icon settings >}} <strong>Settings</strong>.</li>
```
Comment (OVH-Team-Guides) [inline pages/web/web-paas/administration-users/guide.en-asia.md]: Hello @sreedhards, this formatting {{< icon settings >}} won't be rendered when deployed. Could you please remove it ?

Comment (sreedhards) [reply pages/web/web-paas/administration-users/guide.en-asia.md]: Updated

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @sreedhards

**Pull request merged**